Parser/Parser.cs

```diff
@@ -61,6 +62,17 @@ NBlock Block ()
       return vars.ToArray ();
    }
 
+    NConstDecl[] ConstDecl () {
+      List<NConstDecl> vars = new ();
+      while (!Peek (VAR)) {

```

Comment:
Sometimes, there won't be any VAR after const declaration. Your assumption is not right here.